pyrokinetics/pyro.py

```diff
@@ -83,7 +83,7 @@ def gk_code(self, value):
         """
 
         if value is None:
-            self._gk_cde = GKCode()
+            self._gk_code = GKCode()
```

Comment:
THe `GKCode` class on it own doesn't do anything so setting this to `None` should be fine. The only issue is if someone tries to write a file without setting `GKCode` then it will probably fail without a useful warning. Though I think that is the case now so could be a separate issue. Fine with this for now.